docs/rpc.md

```diff
@@ -384,9 +385,14 @@ curl -s -X POST http://127.0.0.1:5000 -H 'Content-Type: application/json' -d \
 Gets information for the specified ongoing swap.
 
 Parameters:
-- `offerID`: the swap's ID.
+- `offerID`: optional: the swap's ID.
 
 Returns:
+- `swaps`: a list of ongoing swaps. If an offerID is provided, this returns only the swap with that ID, if it exists.
+
+Each items in `swaps` contains:
+- `id`: the swap ID.
+- `startTime`: the start time of the swap.
```

Comment:
Should we mention that it is in RFC 3339 format? That's what I did in `swap_suggestedExchangeRate`. I'm not crazy about the way I used parentheses at the end. Feel free to update those entries with whatever style you use here.